src/pages/components/timeline/TimelinePoint.tsx

```diff
@@ -58,7 +58,7 @@ type LabeledPointProps = {
 } & PointProps
 
 export const LabeledPoint = ({ timestamp, top, type }: LabeledPointProps) => {
-  const timeDisplay = timestamp.format(TEXTS.time_format)
+  const timeDisplay = timestamp.format(t('time_format'))
```

Comment:
```suggestion
  const timeDisplay = timestamp.format('time_format')
```

Reply:
Here we can pass the time format directly = HH:mm:ss. If we pass it like 'time_format' this is what we get:
<img width="467" alt="image" src="https://github.com/hasadna/open-bus-map-search/assets/45571546/604e09b4-be52-44a6-91ee-d5ff97ce4e83">

I think I have seen use of the translation modal in another place, for formatting time. If so maybe it should be a constant somewhere.

Reply:
well, it's weird to find it in the translation file, you're right. it can simply be a constant